app/models/job_template.rb

```diff
@@ -65,30 +65,41 @@ def self.base_class
   # Import a template from ERB, with YAML metadata in the first comment.  It
   # will overwrite (sync) an existing template if options[:update] is true.
```

Comment:
could you perhaps add a comment that this method is used by foreman_templates and the API should not change? I don't know if it helps but it's probably worth of trying